src/bin/main.rs

```diff
@@ -94,6 +94,9 @@ struct OutputFormatArg {
     /// Compact output
     #[clap(short, long, conflicts_with = "yaml-output")]
     compact_output: bool,
+    /// Raw string output
+    #[clap(short, long, conflicts_with = "yaml-output")]
+    raw_string_output: bool,
```

Comment:
Could you please rename this to `raw_output` (or change the above attribute to `long = "raw-output"`) so that it matches to that of `jq` and `gojq`?

Reply:
Thank your review. `jq -h` says:
```
  -r               output raw strings, not JSON texts;
```
`-r` does affect only to string output. (ex: `{"foo": true}` or `1` is not)
(I don't have an objection for your opinion. So if you prefer `raw_output`, I'll change this)

Reply:
Ah interesting. Despite it doesn't say it in the help text, the `--raw-output` option actually exists and usable...
https://github.com/stedolan/jq/blob/a9ce7244b4241c457bace5f7d7871268ed00c40b/src/main.c#L358
I think it's better to follow what they do even if it's not documented :)